src/api/client.ts

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const client = axios.create({
+  baseURL: 'http://127.0.0.1:3000',
```

Comment:
@DianaVargasOrellana please extract this hardcoded setting

Reply:
Should I take it out as a constant or put it in an env?

Reply:
Better to an env var

Reply:
I've added webpack dotenv to resolve this